wpeverest/everest-forms
# Fix - Sanitization of CSV exported field values

### All Submissions:






### Changes proposed in this Pull Request:
This PR changes the approach of data formatting in CSV export so that it doesn't affect other tasks like sanitization etc. This PR is associated with this PR: https://github.com/wpeverest/everest-forms-pro/pull/184

### How to test the changes in this Pull Request:

1. Create a form and add `Likert` field and some other fields like `Paragraph Text` etc
2. Make some entries
3. Go to entries and export to CSV
4. View the CSV file in a CSV viewer and see if the line breaks are present for the Likert field value and also see if other field values are affected

### Types of changes:




### Other information:




### Changelog entry

> Fix - Sanitization of CSV exported field values
